### PR TITLE
Translate about.md: "Why we are teaching this class"

### PR DESCRIPTION
**Pull request by Binlogo**
I translated most of the `about.md` but not complete, because this file include many common concepts, so I want to create PR firstly for code(translation) review.

If there is any issues, just comment, I'll do some adjust. @hanxiaomax 

link and solve issue #2 .

### CODE REVIEW

**Comment by Binlogo** [comment]
Sorry for late completion, but it comes here.  😜
Poke @hanxiaomax , wait for review, and hope to accept.

File: about.md [hunk 1]
```diff
@@ -1,141 +1,110 @@
...
-never shown how to use these tools, or at least not how to use them
-efficiently, and thus waste time and effort on tasks that _should_ be simple.
-The standard CS curriculum is missing critical topics about the computing
-ecosystem that could make students' lives significantly easier.
+在传统的计算机科学课程中，从操作系统、编程语言到机器学习，这些高大上课程和主题已经非常多了。
+然而有一个至关重要的主题却很少被专门讲授，而是留给学生们自己去探索。 这部分内容就是：精通工具。
+
+这些年，我们在麻省理工学院参与了许多课程的助教活动，过程当中愈发意识到很多学生对于工具的了解知之甚少。
+计算机设计的初衷就是任务自动化，然而学生们却常常陷在大量的重复任务中，或者无法完全发挥出诸如
+版本控制、文本编辑器等强大作用。效率低下和浪费时间还是其次，更糟糕的是，这还可能导致数据丢失或
```
Comment (hanxiaomax) [inline about.md:11]:
```suggestion
版本控制、文本编辑器等工具的强大作用。效率低下和浪费时间还是其次，更糟糕的是，这还可能导致数据丢失或
```

File: about.md [hunk 2]
```diff
@@ -1,141 +1,110 @@
...
-How to use version control _properly_, and take advantage of it to
-save you from disaster, collaborate with others, and quickly find and
-isolate problematic changes. No more `rm -rf; git clone`. No more
-merge conflicts (well, fewer of them at least). No more huge blocks
-of commented-out code. No more fretting over how to find what broke
-your code. No more "oh no, did we delete the working code?!". We'll
-even teach you how to contribute to other people's projects with pull
-requests!
+如何**正确地**使用版本控制，利用它避免尴尬的情况发生，与他人协作，并且能够快速定位
+有问题的提交改动。不再`rm -rf; git clone`。不再合并冲突（当然，至少是更少发生）。
```
Comment (hanxiaomax) [inline about.md:43]:
```suggestion
如何**正确地**使用版本控制，利用它避免尴尬的情况发生。与他人协作，并且能够快速定位
有问题的提交
```
或者
```suggestion
如何**正确地**使用版本控制，利用它避免尴尬的情况发生。与他人协作，并且能够快速定位
包含有问题改动的提交
```

File: about.md [hunk 3]
```diff
@@ -1,141 +1,110 @@
...
+不再大量注释代码。不再为解决 bug 而找遍所有代码。不再“我去，刚才是删了有用的代码？！”。
+我们将教你如何通过请求合并来为他人的项目贡献代码。
```
Comment (hanxiaomax) [inline about.md:45]:
```suggestion
我们将教你如何通过拉取请求来为他人的项目贡献代码。
```
pull request 暂时统一翻译为拉取请求。

File: about.md [hunk 4]
```diff
@@ -1,141 +1,110 @@
...
 
-## Security
+## 安全
 
-How to be on the Internet without immediately revealing all of your
-secrets to the world. No more coming up with passwords that match the
-insane criteria yourself. No more unsecured, open WiFi networks. No
-more unencrypted messaging.
+如何在不泄露隐私的情况下畅游互联网。不再抓破脑袋想符合自己疯狂规则的密码。不再连接不安全
+的开放 WiFi 网络。不再未加密传输消息。
```
Comment (hanxiaomax) [inline about.md:99]:
```suggestion
的开放 WiFi 网络。不再传输未加密的信息。
```

File: about.md [hunk 5]
```diff
@@ -1,141 +1,110 @@
...
 
-# Conclusion
+# 结论
 
-This, and more, will be covered across the 12 class lectures, each including an
-exercise for you to get more familiar with the tools on your own. If you can't
-wait for January, you can also take a look at the lectures from [Hacker
-Tools](https://hacker-tools.github.io/lectures/), which we ran during IAP last
-year. It is the precursor to this class, and covers many of the same topics.
+以上包括更多的内容将涵盖在 12 节课程中，每堂课都包括能让你自己更熟悉这些工具的练手小测验。如果不能
```
Comment (hanxiaomax) [inline about.md:103]:
```suggestion
这 12 节课将包括但不限于以上内容，同时每堂课都提供了能帮助你熟悉这些工具的练手小测验。如果不能
```
This, and more, will be covered across the 12 class lectures,

and more 做插入语的话，最好还是拿到后面翻译，放在括号里也行。
例如：上述内容都会包含在这十二讲中（当然也不仅限于这些内容）

但是`以上包括更多的内容` 不太通顺

File: about.md [hunk 6]
```diff
@@ -1,141 +1,110 @@
...
 
-## Remote machines
+## 远程服务器
 
-How to stay sane when working with remote machines using SSH keys and
-terminal multiplexing. No more keeping many terminals open just to
-run two commands at once. No more typing your password every time you
-connect. No more losing everything just because your Internet
-disconnected or you had to reboot your laptop.
+使用 SSH 密钥在远程机器下工作如何保持清醒，并且终端能够复用。不再为了仅执行个别命令
```
Comment (hanxiaomax) [inline about.md:66]: stay sane 这里不知道作者想表达的到底是什么，姑且先这样翻译吧

Comment (Binlogo) [reply about.md:66]: 是的，这里确实很难翻，可意会，但不知道怎么同样传神地表达

**Comment by Binlogo** [comment]
> Here are some comments based on my very personal understandings. For your reference

Thanks, nice suggestions, applied.

**Pull request merged**